gpio_controllers/test/test_gpio_command_controller.cpp

```diff
@@ -199,6 +199,9 @@ class GpioCommandControllerTestSuite : public ::testing::Test
 
   std::unique_ptr<FriendGpioCommandController> controller_;
 
+  controller_interface::ControllerInterfaceParams createDefaultParams(
```

Comment:
```suggestion
  controller_interface::ControllerInterfaceParams create_default_params(
```
we use snake case in general. I see that we haven't used that consistently, but please let's use this for every new method.